docs/source/tutorial/01-configure-project.mdx

```diff
@@ -65,7 +65,7 @@ Now add `apollo-runtime` to the list of dependencies. This is the actual code th
 ```groovy:title=app/build.gradle.kts
 dependencies {
     // ...
-    implementation 'com.apollographql.apollo3:apollo-runtime:3.0.0-beta01'
+    implementation("com.apollographql.apollo3:apollo-runtime:3.0.0-beta01")
```

Comment:
this change was to better match how `implementation` is set up in the rest of the gradle file

Reply:
+1 also, it works in both Groovy and Kotlin scripts